binary_search.py

```diff
@@ -0,0 +1,20 @@
+def binary_search(lst, target):
```

Comment:
Mi6: can you please add a docstring

Reply:
Working on it...

*This is an automated message generated by Mi6*